sr_description/meshes/components/th_distal/pst/th_distal_pst.dae

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
  Software License Agreement (BSD License)
- Copyright © 2022 belongs to Shadow Robot Company Ltd.
+ Copyright © 2022-2024 belongs to Shadow Robot Company Ltd.
```

Comment:
In general we should avoid doing this.
This file doesn't need to get touched in this PR.

Reply:
Apparently, this may be due to a glitch in our tests. Because this branch got merged with noetic-devel which has modified this files since..

Reply:
OK